app/models/user.rb

```diff
@@ -3,7 +3,7 @@
 # User: user model
 class User < ApplicationRecord
   audited only: %i[locked_at jurisdiction_id created_at api_enabled role email authy_enabled
-                   force_password_change last_sign_in_with_authy], max_audits: 1000
+                   force_password_change last_sign_in_with_authy notes], max_audits: 1000
```

Comment:
I think this might be kind of expensive, and I don't see an in the audit modal to show this (apologies if I'm missing it). Unless there is a strict requirement to audit notes, let's leave this out.

Reply:
Fixed in https://github.com/SaraAlert/SaraAlert/pull/805/commits/e79c5dfce756880c55693a245c84a72e7208331c